CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ include_directories(${INCLUDE_DIR}
                     ${INCLUDE_DIR}/cimplog
                     ${LIBRARY_DIR}/dbus-1.0/include
                     ${INCLUDE_DIR}/dbus-1.0
+		    ${INCLUDE_DIR}/rbus
```

Comment:
Rbus ExternalProject_Add should also be done in cmakelists.txt

Reply:
added the changes